compose-builder/common-security.env

```diff
@@ -21,5 +21,5 @@
 
 EDGEX_SECURITY_SECRET_STORE=true
 SECRETSTORE_HOST=edgex-vault
-SECRETSTORE_SERVERNAME=edgex-vault
-SECRETSTORE_ROOTCACERTPATH=/tmp/edgex/secrets/ca/ca.pem
+SECRETSTORE_SERVERNAME=''
+SECRETSTORE_ROOTCACERTPATH=''
```

Comment:
Now that these are the defaults, they should be blank in the TOML files and removed from here.

Reply:
removed